src/amo/reducers/versions.js

```diff
@@ -161,7 +201,7 @@ const reducer = (
         bySlug: {
           ...state.bySlug,
           [slug]: {
-            versions: null,
+            versionIds: [],
```

Comment:
Why do you need to initialize this to an empty array? We typically initialize to `null` so the frontend can know for sure that no attempt has been made to fetch the results. In other words, an empty array could be the legitimate result of a fetch.

Reply:
Ah, that's probably why I had it as `null` in the first place. I changed it in the last commit. I'll change it back to `null`.